File: DiplomAPI/Models/Employee.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace DiplomAPI.Models
{
    public partial class Employee
    {
        public int Id { get; set; }
        public string Surname { get; set; }
        public string Name { get; set; }
        public string Middlename { get; set; }
        public DateTime Birthdate { get; set; }
        public string Address { get; set; }
        public string Telephone { get; set; }
    }
}

File: app/src/main/java/com/example/mptfood/Models/Employee.kt
package com.example.mptfood.Models

import android.os.Parcelable
import kotlinx.android.parcel.Parcelize
import java.util.*

@Parcelize
data class Employee
    (
        var ID: Int = 0,
        var Surname: String? = null,
        var Name: String? = null,
        var Middlename: String? = null,
        var Birthdate: Date? = null,
        var Address: String? = null,
        var Telephone: String? = null,

    ) : Parcelable